package.json

```diff
@@ -19,31 +19,84 @@
     "integration-tests:alpha": "node build/testing/sauce-labs/test-browsers.js alpha",
     "integration-tests:beta": "node build/testing/sauce-labs/test-browsers.js beta",
     "integration-tests:release": "node build/testing/sauce-labs/test-browsers.js release",
-    "prettier:diff": "prettier --config .prettierrc '**/*.{ts,tsx,js}' --list-different"
+    "prettier:diff": "prettier --config .prettierrc '**/*.{ts,tsx,js}' --list-different",
```

Comment:
I don't think we're actually running prettier at the root. It's not setup for it currently. I think you can remove this npm script here as it runs via a lerna command across our packages. You can likely remove the tslint-config-prettier as well...unless you're having issues, which would surprise me since we aren't running it here.

Reply:
Got it - removing